project/src/app/app.component.html

```diff
@@ -0,0 +1,56 @@
+<router-outlet>
```

Comment:
remove router-output in this step, we will come back to this tag later when we create routing with components.